FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
I do not understand, fwlite already depends on Xerces via FWCore/Concurrency. So if you want to cleanup that dependency then that is a different issue